packages/market-service/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+// 8 requests per second as per https://www.coingecko.com/en/api_terms section 4.2
+export const COINGECKO_MAX_RPS = 8
```

Comment:
please check the docs again https://www.coingecko.com/en/api/documentation it's 50/min not 8/s

Reply:
hmmm the docs say 50/min but api TOS says 8/s, I will use the 50/min here then as that is the lesser one